pokeapp/src/assets/uiIcons/pokeDollar.svg

```diff
@@ -6,7 +6,7 @@
  preserveAspectRatio="xMidYMid meet">
 
 <g transform="translate(0.000000,1644.000000) scale(0.100000,-0.100000)"
-fill="#fff" stroke="none">
+fill="hsl(0, 0%, 11%)" stroke="none">
```

Comment:
Ah nice, a grey instead of black?  Interested to see how this looks!

Reply:
dark dark grey yes. more contrast